web/conf/dataset.jsonld

```diff
@@ -29,8 +29,8 @@
       "en":"The hbz union catalogue as Linked Open Data"
    },
    "description":{
-      "de":"<p class=\"lead\">Der Verbundkatalog des Hochschulbibliothekszentrums NRW (hbz) enthält die kooperativ erstellten Nachweise von Titel- und Bestandsdaten der Verbundbibliotheken in Nordrhein-Westfalen und Rheinland-Pfalz.</p><p>Daneben sind die Nordrhein-Westfälische Bibliographie und alle Zeitschriften der Zeitschriftendatenbank (ZDB) in den hbz-Verbundkatalog integriert. Die von den Verbundbibliotheken und ihren zugeordneten Einrichtungen aktiv eingebrachten Bestände sind Grundlage für die Online-Fernleihe. Der hbz-Verbundkatalog wird ergänzt um die Bestandsnachweise einer Vielzahl öffentlicher und anderer Bibliotheken, die ihre Bestände maschinell an das hbz liefern.</p><p>Der Verbundkatalog hat einen Umfang von etwa 20 Millionen Titeldatensätzen plus Bestandsinformationen.</p><p>Das hbz, einige Kölner Bibliotheken und das Landesbibliothekszentrum Rheinland-Pfalz starteten 2010 eine Open-Data-Initiative mit der ersten Freigabe von Katalogdaten in Deutschland. Mittlerweile wird der gesamte hbz-Verbundkatalog unter <a href=\"https://creativecommons.org/publicdomain/zero/1.0/\">CC0</a> auf verschiedene Arten bereitgestellt: in dieser Oberfläche, als <a href=\"https://lobid.org/resources/api\">Programmierschnittstelle</a> sowie in Form von Vollabzügen als Linked Open Data und <a href=\"https://lobid.org/download/dumps/DE-605/mabxml/\">MAB/XML</a>. <a href=\"https://lobid.org/resources/dataset\"> Zur Datensetbeschreibung</a>.</p>",
```

Comment:
Better remove the download section. There will be no dump of mab anymore, and maybe even not of MARC at all